AUTHORS.txt

```diff
@@ -71,3 +71,4 @@ Tim Orling                     IronPython support for `Dialogs` library (2.9.2,
 Jozef Behran                   Fix ${TEST_MESSAGE} to reflect current test message (3.0, #2188)
 Joong-Hee Lee                  Extend 'Repeat Keyword' to support timeout (3.0, #2245)
 Anton Nikitin                  Should (Not) Contain Any (3.0.1, #2120)
+Chris Callan                   Case Insensitive 'Should Contain X Times' (3.0.1, #2494)
```

Comment:
No need to add AUTHORS info in all PRs related to same issue. It just causes conflicts.